### ISSUE TITLE
Some type of access warning/error

### ISSUE THREAD

**Issue by ckipp01** [lucasconstantino/now-micro#2]
I tried like 30 builds today to narrow in on what exactly is causing this, but I can't seem to narrow it down to exactly what is causing it -- meaning it may be on Zeit's end. In the beginning I got your example to work fine, but then when I relied on Micro some how directly, for example requiring json, or adding in Micro-dev, things started to fail to build. Then even after removing them and trying to build again, they were consistently failing to build with the below logs.

```
running yarn for now/micro...
yarn add v1.12.3
warning package.json: No license field
info No lockfile found.
warning No license field
[1/4] Resolving packages...
error Command failed.
Exit code: 128
Command: git
Arguments: ls-remote --tags --heads ssh://git@github.com/now/micro.git
Directory: /tmp/utils/build-module
Output:
Could not create directory '/home/sbx_user1059/.ssh'.
Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

```

Where I ended up was with a now.json file like below:

```
{
  "version": 2,
  "name": "micro-test",
  "builds": [
    { "src": "index.js", "use": "now/micro"  }
  ]
}
```
and a package.json file like below:

```
{
  "scripts": {
    "start": "micro",
    "dev": "micro-dev"
  },
  "dependencies": {
    "micro": "^9.3.3",
    "now-micro": "^0.3.0"
  }
}

```

**Comment by lucasconstantino** [comment]
I don't see how can this be related to now-micro. My suggestion is you refactor `index.js` to use `@now/node` and see if the problem happens.

**Comment by ckipp01** [comment]
Well looking at the logs it looks like it's failing when it's running yarn for now-micro. If I build with `@now-micro` the build succeeds. When I add in now-micro, the build fails. I've also been able to replicate this with your example in the readme once I add in the `micro` and `micro-dev` as dependencies and start utilizing their functionality.

**Comment by lucasconstantino** [comment]
Do you mind to publish a full error reproduction repo?

**Comment by ckipp01** [comment]
Of course. [Here is a copy](https://github.com/ckipp01/micro-test) of what I tested and results in the above error when I attempt to publish with `now`.
